docs/04-operation-guides/operations/03-change-kyma-config-values.md

```diff
@@ -26,14 +25,6 @@ You can use the `--values-file` and the `--value` flag.
       istio_pilot:
         version: 1.11.4
         directory: "istio"
-  ory:
```

Comment:
i think this whole document should be reworked since it does not work like that since modularisatio